### PR TITLE
Update info about the Fastly response headers that should be checked for CDN troubleshooting

### PR DESCRIPTION
**Pull request by dmrachkovskyi**
## This PR is a:

- [ ] New topic
- [X] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

This Pull Request updates the **Fastly troubleshooting** topic and specifies the correct list of Fastly response headers that must be checked for Fastly CDN troubleshooting.

### Updates in the "Check response headers" section

This PR removes some of the response headers which, according to the article, must be present in the successful response.

The reason for deleting those headers is the following: the headers removed may only be returned if the request is sent to Fastly, while the request mentioned in this section is directed to the Magento application.

### Updates in the "X-Cache includes MISS" section

This PR renames the section into "X-Cache contains only MISS, no HIT", and explains that the `X-Cashe` tag signals about a problem only when it contains `MISS` and no `HIT`.

A single `HIT`, present in the `X-Cache` tag, means Fastly returns the cached content successfully.

## Additional information

Affected URLs:
- https://devdocs.magento.com/guides/v2.1/cloud/cdn/trouble-fastly.html
- https://devdocs.magento.com/guides/v2.2/cloud/cdn/trouble-fastly.html (symlinked)
- https://devdocs.magento.com/guides/v2.3/cloud/cdn/trouble-fastly.html (symlinked)

whatsnew
Corrected information about [Fastly response headers and values](https://devdocs.magento.com/guides/v2.3/cloud/cdn/trouble-fastly.html ) to verify when troubleshooting  Fastly service issues on Magento Commerce Cloud environments.

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
@dmrachkovskyi thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=3699) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=3699) before we can accept your contribution.<br/>

File: guides/v2.1/cloud/cdn/trouble-fastly.md [hunk 1]
```diff
@@ -207,9 +204,11 @@ If the module is not installed, you need to install in an Integration environmen
 
 During installation and configuration, you should have uploaded the Fastly VCL. These are the base VCL snippets provided by the Fastly module, not custom VCL snippets you create. For instructions, see [Upload Fastly VCL snippets]({{ page.baseurl }}/cloud/cdn/configure-fastly.html#upload-vcl-snippets).
 
-### X-Cache includes MISS {#xcache-miss}
+### X-Cache contains only MISS, no HIT {#xcache-miss}
 
-If `X-Cache` is either `HIT, MISS` or `MISS, MISS`, enter the same `curl` command again to make sure the page wasn't recently evicted from the cache.
+The `X-Cache` that contains `HIT` (`HIT, HIT` or `HIT, MISS`) means Fastly returns the cached content successfully.
```
Comment (meker12) [inline guides/v2.1/cloud/cdn/trouble-fastly.md]:
```suggestion
If the `X-Cache` header contains `HIT` (`HIT, HIT` or `HIT, MISS`), it indicates that Fastly returns the cached content successfully.
```

File: guides/v2.1/cloud/cdn/trouble-fastly.md [hunk 2]
```diff
@@ -207,9 +204,11 @@ If the module is not installed, you need to install in an Integration environmen
...
+
+If `X-Cache` is `MISS, MISS`, and does not contain `HIT`, enter the same `curl` command again to make sure the page wasn't recently evicted from the cache.
```
Comment (meker12) [inline guides/v2.1/cloud/cdn/trouble-fastly.md]:
```suggestion
If the `X-Cache` header is `MISS, MISS` and does not contain `HIT`, run  the `curl` command again to make sure the page was not recently purged from the cache.
```

File: guides/v2.1/cloud/cdn/trouble-fastly.md [hunk 3]
```diff
@@ -123,11 +123,8 @@ For detailed information on hits and misses, see Fastly's [Understanding cache H
 
 Check the returned response headers and values:
```
Comment (meker12) [inline guides/v2.1/cloud/cdn/trouble-fastly.md]:
```suggestion
Verify that the returned response headers and values contain the following content:
```

File: guides/v2.1/cloud/cdn/trouble-fastly.md [hunk 4]
```diff
@@ -123,11 +123,8 @@ For detailed information on hits and misses, see Fastly's [Understanding cache H
...
 
-*	Fastly-Magento-VCL-Uploaded should be present
 *	X-Magento-Tags should be returned
```
Comment (meker12) [inline guides/v2.1/cloud/cdn/trouble-fastly.md]:
```suggestion
*	Includes  `X-Magento-Tags`
```

File: guides/v2.1/cloud/cdn/trouble-fastly.md [hunk 5]
```diff
@@ -123,11 +123,8 @@ For detailed information on hits and misses, see Fastly's [Understanding cache H
...
 *	Fastly-Module-Enabled should be either Yes or the Fastly extension version number
```
Comment (meker12) [inline guides/v2.1/cloud/cdn/trouble-fastly.md]:
```suggestion
*	The `Fastly-Module-Enabled` value is either `Yes` or the version number of the Fastly-Magento module
```

File: guides/v2.1/cloud/cdn/trouble-fastly.md [hunk 6]
```diff
@@ -123,11 +123,8 @@ For detailed information on hits and misses, see Fastly's [Understanding cache H
...
-*	X-Cache should be either `HIT` or `HIT, HIT`
-*	x-cache-hits should be 1,1
 *	[Cache-Control: max-age](https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.9) should be greater than 0
```
Comment (meker12) [inline guides/v2.1/cloud/cdn/trouble-fastly.md]:
```suggestion
*	[Cache-Control: max-age](https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.9) is greater than 0
```

File: guides/v2.1/cloud/cdn/trouble-fastly.md [hunk 7]
```diff
@@ -123,11 +123,8 @@ For detailed information on hits and misses, see Fastly's [Understanding cache H
...
 * [Pragma](https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.32) should be `cache`
```
Comment (meker12) [inline guides/v2.1/cloud/cdn/trouble-fastly.md]:
```suggestion
* [Pragma](https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.32) setting is `cache`
```

**Comment by meker12** [comment]
@dmrachkovskyi  I reviewed the content and added some suggested changes. Let me know if you have questions. I imagine you are waiting on @iskliarenko to review.

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Comment by jeff-matthews** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Comment by dshevtsov** [comment]
@meker12 please use only one label to specify type of changes: `Technical` or `Major update`

Also see suggested changes at #3953

**Pull request merged**